src/main/java/uk/gov/companieshouse/pscfiling/api/config/InterceptorConfig.java

```diff
@@ -21,12 +21,13 @@
 @Configuration
 @ComponentScan("uk.gov.companieshouse.api")
 public class InterceptorConfig implements WebMvcConfigurer {
-
     public static final String COMMON_INTERCEPTOR_PATH =
             "/transactions/{transaction_id}/persons-with-significant-control/{pscType:"
                     + "(?:individual|corporate-entity|legal-person)}";
+    public static final String COMMON_INTERCEPTOR_RESOURCE_PATH =
+            COMMON_INTERCEPTOR_PATH + "/{filing_resource_id}";
     public static final String FILINGS_PATH =
-            "/private" + COMMON_INTERCEPTOR_PATH + "/{filing_resource_id}/filings";
+            "/private" + COMMON_INTERCEPTOR_RESOURCE_PATH + "/filings";
```

Comment:
I can't think, but by modifying this to include the filings resource, does it mean any other requests are no longer going through this interceptor?